src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -260,7 +261,11 @@ int ACA_OVS_L2_Programmer::create_port(const string vpc_id, const string port_na
             cmd_string, culminative_time);
     if (command_rc != EXIT_SUCCESS)
       overall_rc = command_rc;
-  } else {
+  }
+
+  // Disabling this code for testing, as test controller already sets the vlan,
```

Comment:
If it's temp solution for test setup temp, please do not include in PR to master.

Reply:
This one we can leave it as is. As the disabled code is reverted back already. This message can be useful for other people when testing with TC.

Reply:
Fixed